src/Main.java

```diff
@@ -24,57 +24,72 @@ public static void main(String[] args) {
     }
 
     public static void printEmployees (Employee[] employees) {
-        for (int i = 0; i < employees.length - 1 && employees[i] != null; i++) {
-            System.out.println(employees[i]);
+        for (int i = 0; i < employees.length; i++) {
+            if (employees[i] != null) {
+                System.out.println(employees[i]);
+            }
+
         }
     }
 
     public static int calculateSumSalary (Employee[] employees) {
         int sum = 0;
-        for (int i = 0; i < employees.length - 1 && employees[i] != null; i++) {
-            sum = sum + employees[i].getSalary();
+        for (int i = 0; i < employees.length; i++) {
+            if (employees[i] != null) {
+                sum = sum + employees[i].getSalary();
+            }
         }
         return sum;
     }
 
     public static int minSalary (Employee[] employees) {
         int min = employees[0].getSalary();
-        for (int i = 0; i < employees.length - 1 && employees[i] != null; i++) {
-            if (employees[i].getSalary() < min) {
-                min = employees[i].getSalary();
+        for (int i = 0; i < employees.length; i++) {
+            if (employees[i] != null) {
+                if (employees[i].getSalary() < min) {
+                    min = employees[i].getSalary();
+                }
             }
```

Comment:
```suggestion
            if (employees[i] != null && employees[i].getSalary() < min) {
                min = employees[i].getSalary();
            }
```
не нужно создавать вложенность на ровном месте